src/index.js

```diff
@@ -21,12 +25,12 @@ export default class Kinto {
       throw new Error("missing collection name");
 
     const bucket = this._options.bucket || DEFAULT_BUCKET_NAME;
-    const api = new Api(this._options.remote || "http://0.0.0.0:8888/v1", {
+    const api = new Api(this._options.remote || "http://0.0.0.0:8888/v1", this.events, {
```

Comment:
nit not related to the PR, but I think we should use localhost:8000 instead of 0.0.0.0:8000